host_vars/OCT4-SW-TORS/interfaces.yaml

```diff
@@ -0,0 +1,330 @@
+interfaces:
+  1/1:
+    description: "OCT4-17"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/2:
+    description: "OCT4-16"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/3:
+    description: "OCT4-15"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/4:
+    description: "OCT4-14"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/5:
+    description: "OCT4-13"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/6:
+    description: "OCT4-12"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/7:
+    description: "OCT4-11"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/8:
+    description: "OCT4-10"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/9:
+    description: "OCT4-9"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/10:
+    description: "OCT4-8"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/11:
+    description: "OCT4-7"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/12:
+    description: "OCT4-6"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/13:
+    description: "OCT4-5"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/14:
+    description: "OCT4-4"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/15:
+    description: "OCT4-3"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/16:
+    description: "OCT4-2"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/17:
+    description: "OCT4-1"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/18:
+    description: "OCT4-0"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/19:
+    description: "unallocated"
+    admin: "down"
+    mtu: 9216
+  1/20:
+    description: "OCT4-0 2"
+    admin: "up"
+    mtu: 9216
+    untagged: 208
+    tagged: [623]
+  1/21:
+    description: "OCT4-1 2"
+    admin: "up"
+    mtu: 9216
+    untagged: 127
+    tagged: [207]
+  1/22:
+    description: "OCT4-2 2"
```

Comment:
took me a few seconds to realized that this meant the 2nd port. I though it was a typo or something. maybe write `OCT4-2 2nd` or something better to make it clearer.